### ISSUE TITLE
Why not do ReAlign on gsm8k/math

### ISSUE THREAD

**Issue by Connor-Shen** [GAIR-NLP/ReAlign#2]
Very good job! Thank you for your work and dedication！
I have a small question. Have you tried realigning on the gsm8k/math training set and then testing on the test set to see if the performance will improve?
It seems that you only use gsm8k for evaluation and haven't done realign on gsm8k's training set. Kinda of curious why :)

**Comment by RZFan525** [comment]
Thank you for your attention.

I am sorry for my delay. We have reformatted gsm8k and math in our work (see the math reasoning in our paper) but did not upload the ReAlign dataset. 

Now, we update this repo for adding the ReAlign-GSM8K and ReAlign-MATH. You can find these two datasets!

Thank you again!
